### PR TITLE
Fix/refactoring and fixing syntax

### PR DESCRIPTION
**Pull request by okalimbet**
## What is the change?
Fixing syntax by running eslint. Removing unused console.logs, variables, and functions.
Scss files and variables were renamed according to the scss rules.

## What does it fix?
readability, scss naming convention, syntax.

## Is this a fix or a feature?  
Fix

## Where should the reviewer start?

Fixed syntax:

Activity.js: line 5, 
Hydration: line 5, 
Sleep.js: line 1, 
User-repo.js: line 14,
api.request.js: line 2,
index.js: line 11,
script.js: line 56,
fitness-test, activity-test.js, hydration-test.js, sleep-test.js, user-test.js, userrepo-test.js.

Fixing naming convention:
_variables.scss: line 1
_mixins.scss: line 14
styles.scss: line 1

## How should this be tested?
By running npm test should which should show all successfully passed tests. By running npm start the browser should display all random user related information.

We have one issue with the average sleep quality value that needs to be fixed. It shows NaN on the website

### CODE REVIEW

File: src/css/_mixins.scss [hunk 1]
```diff
@@ -32,28 +33,28 @@
...
+  font-size: $font-size-secondary;
+  margin: $margin-primary;
 }
 
 @mixin infocontainer-cardContainer {
-  width: $fullWidth;
-  background-color: $colorHeading;
-  border: $sizeBorder solid $colorBorder;
-  margin: $infoContainerMargin;
+  width: $full-width;
```
Comment (mdflynn) [inline src/css/_mixins.scss:48]: nice job reformatting the scss var styling

File: test/dom-display-test.js [hunk 2]
```diff
@@ -35,27 +36,29 @@ describe('domDisplay', function() {
...
+      'displayRandomUserHistory',
       'addFriendGameInfo',
       'makeFriendChallengeHTML',
       'makeStepStreakHTML'
     ], () => {});
   });
 
   after(function() {
-      chai.spy.restore(domDisplay);
+    chai.spy.restore(domDisplay);
```
Comment (mdflynn) [inline test/dom-display-test.js:47]: Nice use of chai restore.

**Review by mdflynn — APPROVED** [review]
Great job working with the spies.

**Review by cameronRomo — APPROVED** [review]
This all cleaned up very nicely!

**Pull request merged**